server/src/test/java/org/elasticsearch/index/mapper/ExternalFieldMapperTests.java

```diff
@@ -35,12 +33,14 @@ public void testExternalValues() throws Exception {
         assertThat(doc.rootDoc().getField("field.bool"), notNullValue());
         assertThat(doc.rootDoc().getField("field.bool").stringValue(), is("T"));
 
+        /*
```

Comment:
The ExternalFieldMapper tests that values set by a parent field mapper are propagated down to its multifields for a few field types, including a point and shape mapper.  However, no other mapper sets points or shapes as an external field value, so this test was the only place that was using this logic for geo fields.  Given that it is never used in production code and adds a fair amount of complexity to the indexing logic, I've removed it and commented out the tests.

Reply:
Still this is a breaking change so maybe we need to document it accordingly?

Reply:
I've marked it as >breaking and will add notes to the migration docs in the backport PR